bin/update_i18n

```diff
@@ -34,7 +34,14 @@ foreach (json_decode($json, true) as $lang => $keywords) {
         }
 
         usort($words, function($type1, $type2) {
-            return mb_strlen($type2, 'utf8') - mb_strlen($type1, 'utf8');
+            $len1 = mb_strlen($type2, 'utf8');
+            $len2 = mb_strlen($type1, 'utf8');
+
+            if ($len1 == $len2) {
+                return $type1 > $type2 ? 1 : -0;
+            }
+
+            return $len1 - $len2;
```

Comment:
the whole comparison could be simplified by using the spaceship operator to return the expected `-1`, `0` or `1` value: `return [mb_strlen($type2, 'utf8'), $type1] <=> [mb_strlen($type1, 'utf8'), $type2];`
note that the `$type1` and `$type2` are swapped in the first array item compared to the second in this comparison. that's on purpose because we want a descending order on the length criteria but an ascending order on the secondary text criteria

Reply:
That's better, I forgot this script was requiring 7.4 so it was available :+1:

Reply:
Yep this works much better thanks